### PR TITLE
Bugfix/gif no images

### LINKED ISSUE #67: Problem when uploading an animated GIF
Using the CMS and editing the /index page, I tried to set the same image from legacy on the last `category_taxon_shop` block. The image seems to be saved but when I reload the page it is missing. Inspecting the resulting URL the image is unavailable.

Used image: https://d1lfvmh6v1tumg.cloudfront.net/media/cmsimage/639147/original/LeScoop_4Slower.gif?1582563648

### PR DESCRIPTION
**Pull request by KempfCreative**
This fixes the magick arg which was causing the gif to not work. It *does not* implement the resizing with gifsicle, as I have to compile the binary.

Closes #67

### CODE REVIEW

File: lambda/resize.js [hunk 1]
```diff
@@ -59,9 +66,21 @@ exports.default = async (rules, imageVehicle, storageKey, uuid, imageName, tempO
...
+      // const tempGif = [tempOriginalGif, '-resized', '.', ext].join('');
+      // console.log('tempGif ', tempGif);
+      // childProcess.execFile(gifsicle, [tempOriginal, '-o', tempGif, ...gifsicleArgs], (error) => {
+      //   if (error) {
+      //     console.log('gifsicle err ', error);
+      //     return error;
+      //   }
+      //   argsArray = [tempGif, ...magickGifArgs, resizedPath];
+      //   return true;
+      // });
```
Comment (ercgnclvs) [inline lambda/resize.js]: is this supposed to be commented out?

File: lambda/resize.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ const path = require('path');
 const childProcess = require('child_process');
 const fs = require('fs');
 const util = require('util');
+const gifsicle = require('gifsicle');
```
Comment (ercgnclvs) [inline lambda/resize.js]: i dont see this being used? (im slightly blind)

Comment (KempfCreative) [reply lambda/resize.js]: its a WIP.

File: lambda/resize.js (hunk 1 above)
Comment (KempfCreative) [reply lambda/resize.js]: I have it commented out since it should work, but doesn't, and I don't have a commit for it.

**Pull request merged**